testproject/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.12f1
-m_EditorVersionWithRevision: 2020.3.12f1 (b3b2c6512326)
+m_EditorVersion: 2020.3.40f1
```

Comment:
Recalling the conversations in the thread about this PR, there was a mention of this version so I am guessing the manifest and packages-lock updates are based on this?